run/GCHP/ExtData.rc.templates/ExtData.rc.fullchem

```diff
@@ -2151,19 +2151,19 @@ FFF 1 Y Y F2010-%m2-01T00:00:00 none none FFF ./HcoDir/EDGARv43/v2016-11/EDGAR_v
 #==============================================================================
 # --- NEI 2005 scale factors ---
 #==============================================================================
-NEI99_DOW_NOX  1 D N F1999-%m2-01T00:00:00 none none NOX  ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_CO   1 D N F1999-%m2-01T00:00:00 none none CO   ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_ALK4 1 D N F1999-%m2-01T00:00:00 none none ALK4 ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_ACET 1 D N F1999-%m2-01T00:00:00 none none ACET ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_MEK  1 D N F1999-%m2-01T00:00:00 none none MEK  ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_PRPE 1 D N F1999-%m2-01T00:00:00 none none PRPE ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_C3H8 1 D N F1999-%m2-01T00:00:00 none none C3H8 ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_C2H6 1 D N F1999-%m2-01T00:00:00 none none C2H6 ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_SO2  1 D N F1999-%m2-01T00:00:00 none none SO2  ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_SO4  1 D N F1999-%m2-01T00:00:00 none none SO4  ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_MSA  1 D N F1999-%m2-01T00:00:00 none none MSA  ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_BCPI 1 D N F1999-%m2-01T00:00:00 none none BCPI ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
-NEI99_DOW_OCPI 1 D N F1999-%m2-01T00:00:00 none none OCPI ./HcoDir/NEI2005/v2014-09/scaling/NEI99.dow.geos.1x1.nc
+NEI99_DOW_NOX  1 D N F1999-%m2-01T00:00:00 none none NOX  ./HcoDir/NEI2005/v2023-02/scaling/NEI99.dow.geos.1x1.corrected.012023.nc
```

Comment:
File paths also need to be fixed here to remove the scaling/ subdirectory.

Reply:
Will fix